_partials/ubuntu_vscode.md

```diff
@@ -0,0 +1,83 @@
+
+## Visual Studio Code - Your text editor
+
+A text editor is one of the most important tools of a developer.
+Follow these instructions in the Terminal:
+
+```bash
+sudo snap install --classic code
+```
+
+:point_up: This command will ask for your password with: `[sudo] password for <username>:`. Don't panick! Calmly type your password key by key. You won't have a visual feedback (like little `*`), that's **perfectly normal**, keep on typing. When you're done, hit `Enter` :muscle:.
+
+In case you don't have `snap` installed - ask a TA for help.
+
+
+Now let's try to launch your VSCode from **the Terminal**:
+
+```bash
+code
+```
+
+If the text editor opens in a new window, you can move forward!
+
+### VS Code Shortcuts
+
+In VS Code:
+
+>\- Click on `File`
+>\- Click on `Preferences`
+>\- Click on `Keymaps`
+>\- Click on `Sublime Text Keymap and Settings Importer`
+>\- Click on `Install`
+
+
+### VS Code Extensions
```

Comment:
Since this part is going to be common between macOS, Windows and Linux, it should be in a dedicated os-agnostic partial to stay DRY.

Reply:
Good point :pray: